README.md

```diff
@@ -3,58 +3,23 @@
+- [Examples included in the Python client](https://datarobot-public-api-client.readthedocs-hosted.com/en/v2.27.1/examples/index.html)
 
-### MLOps Reporting (Channel Infrastructure, Drift Tracking, Accurracy/Results Reporting)
-[Model Monitoring with Serverless MLOps Agents](https://community.datarobot.com/t5/resources/model-monitoring-with-serverless-mlops-agents/ta-p/7147) - Monitoring externally deployed models with Lambda serverless agents reporting MLOps results \
-[MLOps Agent with GKE and Pub/Sub](https://community.datarobot.com/t5/resources/mlops-agent-with-gke-and-pub-sub/ta-p/9649) - Creating a MLOps reporting channel and monitoring agent to report data to DR in GCP \
-[AWS Lambda Serverless Reporting Actuals to DataRobot MLOps](https://community.datarobot.com/t5/resources/aws-lambda-serverless-reporting-actuals-to-datarobot-mlops/ta-p/9999) - Actuals reporting back to DataRobot based on serverless trigger architecture \
-[Deploy in SageMaker and Monitor with MLOps Agents](https://community.datarobot.com/t5/resources/deploy-in-sagemaker-and-monitor-with-mlops-agents/ta-p/5771) - Monitoring a DataRobot java scoring code model from within a SageMaker container \
-[Monitoring a SageMaker Deployed Model in DataRobot MLOps](https://community.datarobot.com/t5/resources/monitoring-a-sagemaker-deployed-model-in-datarobot-mlops/ta-p/9591) - Monitoring a SageMaker developed model hosted on a SageMaker real time endpoint \
-[Measuring Prediction Accuracy: Uploading Actual Results](https://community.datarobot.com/t5/resources/measuring-prediction-accuracy-uploading-actual-results/ta-p/7907) - Uploading actual results to DataRobot related to predictions 
+## Important Links
 
-### Portable Prediction Server (PPS) - DataRobot Models in Docker Images Deployed as Containers within Kubernetes
-[Deploy and Monitor DataRobot Models in AKS](https://community.datarobot.com/t5/resources/deploy-and-monitor-datarobot-models-in-aks/ta-p/7317) - Deploy DataRobot model (PPS) as docker container on Azure AKS \
-[Deploy and Monitor DataRobot Models on Google Cloud Platform](https://community.datarobot.com/t5/resources/deploy-and-monitor-datarobot-models-on-google-cloud-platform/ta-p/7675) - Deploy DataRobot model (PPS) as docker container on GCP GKE \
-[Deploy and Monitor DataRobot Models on AWS](https://community.datarobot.com/t5/resources/deploy-and-monitor-datarobot-models-on-aws/ta-p/8762) - Deploy DataRobot model (PPS) as docker container on AWS EKS \
-[Path-based routing to Portable Prediction Servers on AWS](https://community.datarobot.com/t5/resources/path-based-routing-to-portable-prediction-servers-on-aws/ta-p/9093) - Routing PPS docker container model requests through a single host/IP entry point 
+- To learn to use DataRobot, visit [DataRobot University](https://university.datarobot.com/).
+- For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/).
```

Comment:
please change:
- For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/).
to:
- To ask questions and get help from DataRobot experts and peers, visit [DataRobot Community](https://community.datarobot.com/)

(that should be the same change you made for the two other repos)

Reply:
```suggestion
- For articles on using DataRobot, feature deep dives, and example workflows, visit [DataRobot Community](https://community.datarobot.com/).
to:
- To ask questions and get help from DataRobot experts and peers, visit [DataRobot Community](https://community.datarobot.com/)
```